mix.exs

```diff
@@ -14,13 +16,21 @@ defmodule Wallaby.Mixfile do
      description: "Concurrent feature tests for elixir",
      deps: deps(),
      docs: docs(),
-     test_coverage: [tool: ExCoveralls],
+
+     # Custom testing
+     aliases: ["test.all": ["test", "test.drivers"],
+               "test.drivers": &test_drivers/1],
      preferred_cli_env: [
-       "coveralls": :test, "coveralls.detail": :test, "coveralls.post": :test,
-       "coveralls.html": :test, "coveralls.travis": :test
-     ],
-    dialyzer: [plt_add_apps: [:inets]]
-   ]
+       "coveralls": :test,
+       "coveralls.detail": :test,
+       "coveralls.post": :test,
+       "coveralls.html": :test,
+       "coveralls.travis": :test,
+       "test.all": :test,
+       "test.drivers": :test],
```

Comment:
It looks like I can remove this line and things will still work when I run `mix test.drivers` (to test all drivers). With this line removed, `mix test.drivers` starts up in the `:dev` environment. Then when `mix test.drivers` spawns each of the driver test suites using `mix test`, those processes will run in the `:test` environment (since they are separate OS processes). I feel like it's kind of unexpected that `mix test.drivers` would run in the `:dev` environment while kicking off tests in that run in the `:test` environment, but it does work. I'm happy to remove this line if you'd like.